core/src/main/java/org/nervos/ckb/utils/Calculator.java

```diff
@@ -0,0 +1,27 @@
+package org.nervos.ckb.utils;
+
+import java.math.BigInteger;
+import org.nervos.ckb.type.dynamic.Table;
+import org.nervos.ckb.type.transaction.Transaction;
+
+/** Copyright © 2019 Nervos Foundation. All rights reserved. */
+public class Calculator {
+  private static final int SERIALIZED_TX_OFFSET_BYTE_SIZE = 4;
+
+  public static int calculateSerializedSizeInBlock(Transaction transaction) {
+    Table serializedTx = Serializer.serializeTransaction(transaction);
+    return serializedTx.getLength() + SERIALIZED_TX_OFFSET_BYTE_SIZE;
+  }
+
+  private static final BigInteger RADIO = BigInteger.valueOf(1000);
+
+  public static BigInteger calculateMinTransactionFee(
```

Comment:
Can it be `calculateTransactionFee(BigInteger transactionSize, BigInteger feeRate)` ?

Reply:
Updated. https://github.com/nervosnetwork/ckb-sdk-java/pull/190/commits/350061a9cc7e1b35ce0cd4af43e06d6b31a0466c